public/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" mode="ios">
```

Comment:
is this intended?

Reply:
yes, this is to keep iOS by default for. the browser.